tools/ci.yml

```diff
@@ -16,14 +15,19 @@ services:
     image: data61/anonlink-app:${TAG:-latest}
     environment:
       - SERVER=http://nginx:8851
-      - INITIAL_DELAY=20
```

Comment:
you removed the env var `INITIAL_DELAY` in a couple of places. Who uses this? What is it defaulting to now?

Reply:
It is an optional environment variable used by both the integration tests and end to end tests - I probably shouldn't have deleted them as they serve as documentation.

Note I also added a small sleep in the docker `command` for the integrationtests container